packages/expo-stories/server/writeStoriesJSFile.ts

```diff
@@ -0,0 +1,67 @@
+import fs from 'fs';
+import path from 'path';
+
+import { IServerConfig, IStoryManifestItem } from '../types';
+import { getStories, getStoriesCacheDir } from './shared';
+
+function writeStoriesJSFile(serverConfig: IServerConfig) {
+  const stories = getStories(serverConfig);
+
+  let template = `
+      const storiesToExport = {}
+      ${writeStoryRequires(stories)}
+      module.exports = storiesToExport
+    `;
```

Comment:
FWIW, this creates a file with number of whitespaces at the beginning of each line 😉
Possibly this is not important at all 😅 
```suggestion
      const storiesToExport = {}
      ${writeStoryRequires(stories)}
      module.exports = storiesToExport
    `;
```